modules/mapbox/src/deck-utils.ts

```diff
@@ -112,11 +112,8 @@ export function removeDeckInstance(map: Map & {__deck?: Deck | null}) {
 }
 
 export function getInterleavedProps(currProps: DeckProps) {
-  const useDevicePixels = 'useDevicePixels' in currProps ? currProps.useDevicePixels : true;
-
   const nextProps: DeckProps = {
     ...currProps,
-    useDevicePixels,
```

Comment:
Resolution can currently be controlled in two parts. One, by setting the width/height css style of the react-map-gl HTML container, and two, multiplying width/height by setting `useDevicePixels` prop on deck and `pixelRatio` prop on react-map-gl for maplibre. 

Was this redundant or does this change modify behavior?

Reply:
You should be able to just set `pixelRatio` on the base map after this PR.